pom.xml

```diff
@@ -112,7 +122,15 @@
 					<cleanDisabled>false</cleanDisabled>
 				</configuration>
 			</plugin>
-		</plugins>
+            <plugin>
+                <groupId>org.apache.maven.plugins</groupId>
+                <artifactId>maven-compiler-plugin</artifactId>
+                <configuration>
+                    <source>16</source>
```

Comment:
shouldn't this be 17?

Reply:
I'm honestly not sure but I can change it to 17